### PR TITLE
Add Seravo.com to hosting providers

### PR DESCRIPTION
**Pull request by ottok**
Add Seravo.com to directory at https://certbot.eff.org/hosting_providers

### CODE REVIEW

**Comment by ottok** [comment]
Ping @joohoi - please merge this.

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -1619,6 +1619,16 @@
...
   },
+  {
+    "name": "Seravo",
+    "link": "https://seravo.com",
+    "category": "full",
+    "tutorial": "https://help.seravo.com/article/24-how-do-i-enable-the-https-on-our-website",
+    "announcement": "https://seravo.com/blog/https-is-not-optional/",
+    "plan": "",
+    "reviewed": "2020.11.09",
+    "note": "HTTPS is mandatory for sites at Seravo. All domains have automatically Let's Encrypt certificates and they are enabled by default."
```
Comment (ohemorange) [inline _data/hosting_providers.json:1630]: The note field is for things like "this language is in German".
```suggestion
    "note": ""
```

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -1619,6 +1619,16 @@
     "reviewed": "",
     "note": ""
   },
+  {
+    "name": "Seravo",
+    "link": "https://seravo.com",
+    "category": "full",
+    "tutorial": "https://help.seravo.com/article/24-how-do-i-enable-the-https-on-our-website",
```
Comment (ohemorange) [inline _data/hosting_providers.json:1626]: It looks like the user has to manually specify that they want https. What happens if the user puts in an http address?

**Review by ohemorange — CHANGES_REQUESTED** [review]

File: _data/hosting_providers.json (hunk 1 above)
Comment (ottok) [reply _data/hosting_providers.json]: Removed.

File: _data/hosting_providers.json (hunk 2 above)
Comment (ottok) [reply _data/hosting_providers.json:1626]: HTTPS is mandatory for sites at Seravo. All domains have automatically Let's Encrypt certificates and they are enabled by default. There is nothing the users need to do, unless they have first done something stupid, like manually changed stuff in their database to have http instead https.

Maybe I should just remove the tutorial link? There is nothing customers normally need to do, HTTPS is always on and mandatory.

Comment (ohemorange) [reply _data/hosting_providers.json:1626]: Ah, I see, thank you for clarifying! We only show either the "announcement" or the "tutorial", so yes, removing the tutorial link is the best call.

**Review by ohemorange — CHANGES_REQUESTED** [review]

**Comment by ottok** [comment]
OK, link to tutorial removed as it is unnecessary.

**Review by ohemorange — APPROVED** [review]
Thank you!

**Pull request merged**